python/server.py

```diff
@@ -197,123 +67,18 @@ def classify_one(id, uri=None):
-          "showFungible": False,
-          "showInscription": False,
-        },
-      },
-    },
-  )
-  rpcResponse = response.json()
-
-  if "error" in rpcResponse:
-    return "error"
-
-  compressionData = rpcResponse["result"]["compression"]
-  treeId = compressionData["tree"]
-  proofLength = get_proof_length(treeId)
-
-  # prefer cdn when available
-  # imageUrl = ""
-  # if "cdn_uri" in rpcResponse["result"]["content"]["files"][0]:
-  #   imageUrl = rpcResponse["result"]["content"]["files"][0]["cdn_uri"]
-  # else:
-  #   imageUrl = rpcResponse["result"]["content"]["links"]["image"]
-
-  if ("image" in rpcResponse["result"]["content"]["links"]):
-    imageUrl = rpcResponse["result"]["content"]["links"]["image"]
-    imageWords = get_image_words(imageUrl)
-  else:
-    imageWords = []
-
-  attributeWords = []
-
-  if ("attributes" in rpcResponse["result"]["content"]["metadata"]):
-    attributes = rpcResponse["result"]["content"]["metadata"]["attributes"]
-    for attribute in attributes:
-      if ("value" in attribute):
-        attributeWords += str(attribute["value"]).split()
-      if ("trait_type" in attribute):
-        attributeWords += str(attribute["trait_type"]).split()
-
-  tokens = imageWords + attributeWords
-
-  keywords = [
-    "containsEmoji",
-    "proofLengthImpossible",
-    "imageContainsUrl",
-    "not_containsEmoji",
-    "not_proofLengthImpossible",
-    "not_imageContainsUrl",
-  ]
-
-  tokens = list(filter(lambda token: len(token) > 2 and token not in keywords, tokens))
-
-  EMOJI_PATTERN = re.compile(
-    "["
-    "\U0001F1E0-\U0001F1FF"  # flags (iOS)
-    "\U0001F300-\U0001F5FF"  # symbols & pictographs
-    "\U0001F600-\U0001F64F"  # emoticons
-    "\U0001F680-\U0001F6FF"  # transport & map symbols
-    "\U0001F700-\U0001F77F"  # alchemical symbols
-    "\U0001F780-\U0001F7FF"  # Geometric Shapes Extended
-    "\U0001F800-\U0001F8FF"  # Supplemental Arrows-C
-    "\U0001F900-\U0001F9FF"  # Supplemental Symbols and Pictographs
-    "\U0001FA00-\U0001FA6F"  # Chess Symbols
-    "\U0001FA70-\U0001FAFF"  # Symbols and Pictographs Extended-A
-    "\U00002702-\U000027B0"  # Dingbats
-    "\U000024C2-\U0001F251" 
-    "]+",
-  flags = re.UNICODE) 
-
-  containsUrl = False
-  containsEmoji = False
-
-  for token in tokens:
-    if (re.search(r'^[\S]+[.][\S]', token)):
-      containsUrl = True
-    if (re.search(EMOJI_PATTERN, token)):
-      containsEmoji = True
-
-  if (proofLength > 23):
-    tokens.append("proofLengthImpossible")
-  else:
-    tokens.append("not_proofLengthImpossible")
-
-  if (containsUrl):
-    tokens.append("imageContainsUrl")
-  else:
-    tokens.append("not_imageContainsUrl")
-  
-  if (containsEmoji):
-    tokens.append("containsEmoji")
-  else:
-    tokens.append("not_containsEmoji")
-
+  tokens = extract_tokens(token_id, rpcUrl)
   classification = classify(tokens)
 
   cnftTable.put_item(
     Item={
-      'address': id,
+      'address': token_id,
```

Comment:
changing the id reserved keyword to token_id

Reply:
so that's what the special syntax highlighting meant